zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/SlackNotification.java

```diff
@@ -27,88 +43,163 @@
  */
 public class SlackNotification {
 
+    public enum CollectionStage {
+        PrePublish,
+        Publish,
+        PostPublish
+    }
+
+    public enum StageStatus {
+        Started,
+        Completed,
+        Failed
+    }
+
     private static final String slackToken = System.getenv("slack_api_token");
     private static final String slackDefaultChannel = System.getenv("slack_default_channel");
     private static final String slackAlarmChannel = System.getenv("slack_alarm_channel");
     private static final String slackPublishChannel = System.getenv("slack_publish_channel");
 
-    private static final String slackBaseUri = "https://slack.com/api/chat.postMessage";
-    private static final Host slackHost = new Host(slackBaseUri);
+    private static final String slackPostMessageURI = "https://slack.com/api/chat.postMessage";
```

Comment:
I know it was already like this but if you are making changes to do it properly then we should probably make this config value.

Reply:
I'm not sure it's worth it for these, there's no alternative URLs we could ever use?